subsys/net/lib/coap/Kconfig

```diff
@@ -0,0 +1,43 @@
+# Kconfig - Coap, CoAP implementation for Zephyr
+
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config COAP
+	bool
+	prompt "CoAP Support"
+	default n
+	help
+	This option enables the implementation of CoAP.
+
+# This setting is only used by unit test. Do not enable it in applications
+config COAP_TEST_API_ENABLE
```

Comment:
Not really related to this commit, but noticed that this option should be made hidden so that it would not be shown to the user at all.